src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -233,7 +233,11 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
             ]),
             p({}, [
               'If you think the workspace exists but you do not have access, please contact the workspace owner.'
-            ])
+            ]),
+            buttonPrimary({
+              as: 'a',
+              href: Nav.getLink('workspaces')
+            }, ['Workspaces List'])
```

Comment:
What about naming the button `Return to Workspaces`? I think it makes it a clearer call-to-action.

Reply:
Brad's recommendation was "Return to Workspace List"